vendor-extra/JatsContentBundle/src/EventListener/ItemTagsListener.php

```diff
@@ -20,20 +19,22 @@ public function __construct(ViewConverter $converter)
         $this->converter = $converter;
     }
 
-    public function onCreatePage(CreateContentPageEvent $event) : void
+    public function onCreatePageMain(CreateContentPagePartEvent $event) : void
     {
         $front = $event->getItem()->xpath()->firstOf('/libero:item/jats:article/jats:front');
 
         if (!$front instanceof Element) {
             return;
         }
 
-        $itemTags = $this->converter->convert($front, '@LiberoPatterns/item-tags.html.twig', $event->getContext());
+        $context = ['area' => 'primary'] + $event->getContext();
```

Comment:
`primary` as a term will be part of a very popular interface, what are the alternatives for this value (when the `area` is not `primary`)? Is it standard teminology from Twig or another library?

Reply:
Refers to the area in the grid pattern: https://github.com/libero/pattern-library/blob/80b20661bb84aba1dbdc533bfcd5e13a3c272337/source/css/sass/patterns/templates/content-grid.scss#L69-L88

Reply:
Related: b66733c